### PR TITLE
Add manifest for PathWave License Manager 2.7.0

### PR DESCRIPTION
**Pull request by rkeithhill-keysight**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
 We are still on manifest version 1.1.0. Do we need to update to 1.2.0?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82473)

### CODE REVIEW

**Comment by rkeithhill-keysight** [comment]
Please hold off on merging this PR until I post back.  Nothing wrong with the installer/product - just waiting on one last internal approval to release.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**
